Mathlib/LinearAlgebra/RootSystem/Finite/Lemmas.lean

```diff
@@ -231,6 +231,27 @@ lemma root_add_root_mem_of_pairingIn_neg (h : P.pairingIn ℤ i j < 0) (h' : α
   replace h' : i ≠ -j := by contrapose! h'; simp [h']
   simpa using P.root_sub_root_mem_of_pairingIn_pos h h'
 
+omit [Finite ι] in
+lemma root_mem_submodule_iff_of_add_mem_invtSubmodule
+    {K : Type*} [Field K] [NeZero (2 : K)] [Module K M] [Module K N] {P : RootPairing ι K M N}
+    (q : Submodule K M) (hq : ∀ i, q ∈ Module.End.invtSubmodule (P.reflection i))
```

Comment:
Should `hq` us the `invtRootSubmodule` from the other PR?

Reply:
Yes: on reflection, I think this is likely to be a more useful spelling.